src/test/java/chapter5/Question4Test.java

```diff
@@ -0,0 +1,21 @@
+package chapter5;
+
+import org.junit.jupiter.api.Test;
+import src.chapter5.question4.Question4;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class Question4Test {
```

Comment:
Two test scenarios are missing: null and empty array as parameter.

Reply:
Test scenarios have been added.